src/usability-testing/usability-testing.tex

```diff
@@ -43,90 +41,82 @@ \section{Participants}\label{sec:participants}
+moves along with the current move.
+None of them thought that the feedback was helpful.
+This indicates that the feedback from Stockfish has a lot of room for improvement, as only providing the best move is
+not enough for learning, because players wouldn't understand why it's the best move.
+The team was already aware of this issue and had discussed a potential solution, which was conveyed to the users in the
+next focus point.
 
-Clara also mentioned that the game should have more teaching tools, such as puzzles and guides to how to play chess.
-The team considered these features given the learning nature of the game, but there are already existing solutions that
-provide them, so they were not prioritized, instead focusing on the coaching aspect of the game.
+\subsection{Potential solution}\label{subsec:potential-solution}
 
-Lastly, the users were asked about the idea of using machine learning along with Stockfish to provide feedback.
+As a potential solution, users were asked about the idea of using a large language model along with Stockfish to provide
+feedback.
 The rationale behind this question was due to a prior discussion within the team, which will be later discussed in
 Section~\ref{sec:large-language-model-implementation}.
-Clara and Ronja both liked the idea of using machine learning, while Peter was more skeptical.
+Clara and Ronja both liked the idea of using a large language model, while Peter was more skeptical.
 He did not particularly like the idea due to the lack of control over the large language model's answers, so the team
 cannot be guaranteed that it'll provide correct information.
 Clara however thinks that in combination with Stockfish it could provide accurate information whilst also giving more
 natural responses and explanations.
+She and Ronja think that a natural response would help a lot with understanding the feedback.
+An example given was that the player could ask follow-up questions, which the large language model could answer,
+providing a more interactive learning experience.
 
-% textidote: ignore begin % due to section being too short
+\subsection{Other feedback}\label{subsec:other-feedback}
 
-\subsection{Feedback}\label{subsec:feedback}
-% textidote: ignore end
+In terms of visual feedback, Peter pointed out that the front page was not intuitive and Clara suggested that the
+buttons should be larger, labelled, and link together.
+Peter also mentioned that it was difficult to figure out how to play, he proposed that the About page should contain
+information on how to create an account and what the website is about.
+The in-game interface was also critiqued.
+Clara noted that the Stockfish feedback lacked visual cues, her idea was that there should be arrows on the chessboard,
+which is something that the team had previously discussed.
+Peter mentioned that the notations provided by Stockfish were incorrect and complained about the lack of sound effects
```

Comment:
Maybe say "over complicated" instead of "incorrect", as they are correct, but are not simplified as they are on Chess.com